theano/gpuarray/__init__.py

```diff
@@ -58,6 +58,8 @@ def pygpu_parse_version(version_string):
 
 def init_dev(dev, name=None, preallocate=None):
     global pygpu_activated
+    if os.environ.get('THEANO_GPU_IS_ALREADY_ACTIVE', '') == 'Yes':
```

Comment:
I forgot, why we can import theano, but not the GPU in the subprocess?

Otherwise, I'm good with this PR. I restarted a travis run that failed due to a 10m timeout.